HeliPort/Appearance/StatusMenu.swift

```diff
@@ -409,7 +410,7 @@ final class StatusMenu: NSMenu, NSMenuDelegate {
     // - MARK: Action handlers
 
     @objc private func clickMenuItem(_ sender: NSMenuItem) {
-        Log.debug("Clicked \(sender.title)")
+        Log.debug("Clicked \(sender.title))")
```

Comment:
Extra Bracket?

Reply:
Yup, I missed that! I was trying to undo the localization so it can be in english in the log, but I couldn't figure out to get the key from the value.